static/templates/settings/alert_word_settings.hbs

```diff
@@ -30,11 +30,11 @@
     <div class="progressive-table-wrapper" data-simplebar>
         <table class="table table-condensed table-striped wrapped-table">
             <thead>
-                <th class="active">{{t "Word" }}</th>
+                <th data-sort="alphabetic" data-sort-prop="word">{{t "Word" }}</th>
                 <th>{{t "Actions" }}</th>
             </thead>
-            <tbody id="alert_words_list" class="alert_words_list required-text thick"
-              data-empty="{{t 'There are no current alert words.' }}"></tbody>
+            <tbody id="alert-words-table" class="alert-words-table required-text thick"
+              data-empty="{{t 'There are no current alert words.' }}" data-list-widget="alert-words-list"></tbody>
```

Comment:
This `data-list-widget` field can be removed too, right?

Reply:
No, This `data-list-widget` helps us to identify the `tbody` which we want to render in through `ListWidget` in `alert_words_ui.js` 

here's the code used for it- 
`    ListWidget.create(word_list, words, {
        name: "alert-words-list",
        modifier(alert_words) {
            return render_alert_word_settings_item({alert_words});
        },
        parent_container: $("#alert-word-settings"),
        simplebar_container: $("#alert-word-settings .progressive-table-wrapper"),
    });`

Reply:
have a look at `name` under `Listwidget.create()` to which we render the `tr` present in `alert_word_settings_item`.

Reply:
though the functionality would work without its presence also, but i think it's better to keep it visible in the DOM also, so that it's looks/easier to understand the code written.

Reply:
No, `data-list-widget` is not used anywhere. We use the `$container` element that you pass as first parameter in create. See list_widget.js for clarity. And also you can see we do not use `data-list-widget` at other places as well like in default streams. It is only used for muted topics and muted users section. It was used before but was then removed.

Reply:
Hmmmm, this was confusing. thanks for the clarification @sahil839 .

Reply:
Updated the PR.